portfolio/src/main/webapp/blogpage.css

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: auto;
+  font-family: 'Montserrat', serif;
+}
+
+.landing-section {
+  background-color: cyan;
+  height: 100vh;
+  width: 100vw;
+}
+
+.title {
+  height: 55vh;
+  width: 100vw;
+}
+
+.title h1 {
+  font-size: 60pt;
+  padding: 20vh 0 20vh 0;
+}
+
+#rect {
+  width: 100vw;
+  height: 45vh;
+  background-color: black;
+}
+
+#pole {
+  background-color: red;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  width: 3vw;
+  height: 350vh;
+  margin-left: 49vw;
+}
+
+.blog-section {
+  height: 150vh;
+  width: 100vw;
+  margin-top: -350vh;
+}
+
+.blog-grid {
+  margin-top: 10vh;
+  flex-wrap: wrap;
+}
+
+.blog {
+  width: 37.5vw;
+  height: 67vh;
+  border: 2px solid blue;
+  padding: 0 1.5vw 0 1.5vw;
+}
+
+.thumbnail img {
+  max-width: 100%;
+  width: 100%;
```

Comment:
setting both max-width and width to 100% is redundant - you should just need width here!